doc/library/xtensor/type.md

```diff
@@ -0,0 +1,30 @@
+(libdoc_xtenor_type)=
+
+# `xtensor.type` -- Types and Variables
+
+## XTensorVariable creation functions
+
+```{eval-rst}
+.. currentmodule:: pytensor.xtensor.type
+.. autosummary::
+   :toctree: generated/
```

Comment:
similar to previous comment, I might remove the toctree generated (keep autosummary here though) so all the pages have the same look